tests/testthat/test-save_e61.R

```diff
@@ -188,145 +211,264 @@ test_that("Does save_data work", {
+  data <- data.frame(
+    x = seq.Date(as.Date("2011-01-01"), by = "1 year", length.out = 10),
+    y = runif(10, -1, 1),
+    ymin = runif(10, -2, -1.1),
+    ymax = runif(10, 1.1, 2)
+  )
 
-  rstudioapi::viewer(save_e61(withr::local_tempfile(fileext = ".svg"), mp))
+  p <- ggplot(data, aes(x, y, ymin = ymin, ymax = ymax)) +
+    geom_line() +
+    geom_ribbon(alpha = 0.1)
 
-  # No title, no subtitle, footnotes, sources
-  mp <- mpanel_e61(graph, graph,
-                   footnotes = "Footnote 1",
-                   sources = "Source 1")
+  expect_snapshot_file(suppressWarnings(save_e61("plot-single-cont-ymin-max.svg", p)))
 
-  rstudioapi::viewer(save_e61(withr::local_tempfile(fileext = ".svg"), mp, height = 8.3))
+  # geom_histogram graph
+  data <- data.frame(x = rnorm(1000))
 
-  # Title, subtitle, footnotes, sources
-  mp <- mpanel_e61(graph, graph,
-                   title = "Title",
-                   subtitle = "Graph subtitle",
-                   footnotes = "Footnote 1",
-                   sources = "Source 1")
+  p <- ggplot(data, aes(x)) +
+    geom_histogram()
 
-  rstudioapi::viewer(save_e61(withr::local_tempfile(fileext = ".svg"), mp))
+  expect_snapshot_file(suppressWarnings(save_e61("plot-single-hist.svg", p)))
 
-  # 2x2 with title, subtitle, footnotes, sources
-  mp <- mpanel_e61(graph, graph, graph, graph,
-                   ncol = 2,
-                   title = "Title",
-                   subtitle = "Graph subtitle",
-                   footnotes = "Footnote 1",
-                   sources = "Source 1",
-                   show_height = TRUE)
+  # geom_density graph
+  p <- ggplot(data, aes(x)) +
+    geom_density()
 
-  rstudioapi::viewer(save_e61(withr::local_tempfile(fileext = ".svg"), mp))
+  expect_snapshot_file(suppressWarnings(save_e61("plot-single-cont-dens.svg", p)))
 
 })
 
+test_that("Multi-panel graph examples", {
+
+  withr::local_options(list(test_save = TRUE,
+                            quiet_wrap = TRUE))
+
+  withr::local_seed(42)
+
+  # Graphs to use in the panels
+
+  # Graph with cont-y var with small values from 0-1
+  data <- data.frame(x = factor(1:10), y = runif(10, 0, 1))
+
+  p1 <- ggplot(data, aes(x, y)) +
+    geom_col()
+
+  p1_t <- p1 +
+    labs_e61(title = "Panel graph title text",
+             subtitle = "Panel graph subtitle text",
+             y = "ppt")
+
+  # Graph with date x var and colour
+  data <- data.frame(
+    x = rep(seq.Date(as.Date("2011-01-01"), by = "1 year", length.out = 10), 2),
+    y = runif(20, -1, 1),
+    colour = c(rep("A", 10), rep("B", 10))
+  )
+
+  p2 <- ggplot(data, aes(x, y, colour = colour)) +
+    geom_line()
+
+  p2_t <- p2 +
+    labs_e61(title = "Panel graph title text",
+             subtitle = "Panel graph subtitle text",
+             y = "'000")
+
+  # Graph with cont-y var with large values from -1000 to +1000
+  data <- data.frame(x = factor(1:10), y = runif(10, -1000, 1000))
+
+  p3 <- ggplot(data, aes(x, y)) +
```

Comment:
@JackBuckley not sure why this graph is not generating correctly in the single panel or multi-panel

Reply:
Fixed this now